data/library/tlg1271/tlg002/metadata.json

```diff
@@ -14,7 +14,7 @@
       "urn": "urn:cts:greekLit:tlg1271.tlg002.oaf-1:",
       "node_kind": "version",
       "version_kind": "edition",
-      "first_passage_urn": "urn:cts:greekLit:tlg1271.tlg002.oaf-1:0-1",
+      "first_passage_urn": "urn:cts:greekLit:tlg1271.tlg002.1st1K-grc1:1",
```

Comment:
this needs to be `oaf-1` with my changes

Reply:
Good catch, fixing that up now.